[14.0] account_cutoff_accrual_picking: add prepaid support

alexis-via opened the pull request:
The module should be renamed from account_cutoff_accrual_picking to account_cutoff_picking in v16, now that it has prepaid expense/revenue support.

alexis-via committed:
account_cutoff_accrual_picking: add prepaid support

The module should be renamed from account_cutoff_accrual_picking to account_cutoff_picking in v16, now that it has prepaid expense/revenue support.

tv-openbig commented:
@alexis-via I have checked all four cut-off cases with proper accounts (i have used our typical german accounts for that). For all cases i get the expected results from an accountant perspective. Thanks a lot for adding the prepaid expenses / revenues.
![Bildschirmfoto 2023-01-16 um 10 03 04](https://user-images.githubusercontent.com/8348598/212640638-3deb6b3c-0f96-49dd-87b3-35e7d7632ce8.png)

alexis-via committed:
[FIX] account_cutoff_accrual_picking: dropship

account_cutoff_accrual_picking now works well in the dropship scenario
use native method _get_outgoing_incoming_moves()

alexis-via committed:
[FIX] readonly status of account.cutoff fields

simahawk commented:
can we have some test coverage?

simahawk commented:
BTW you can rebase: I added GH actions to branch 14.0 here https://github.com/OCA/account-closing/pull/233

alexis-via commented:
/ocabot merge major
